src/beast/beast/utility/impl/PropertyStream.cpp

```diff
@@ -192,37 +193,31 @@ std::string const& PropertyStream::Source::name () const
 
 void PropertyStream::Source::add (Source& source)
 {
-    SharedState::Access state (m_state);
-    SharedState::Access childState (source.m_state);
-    bassert (childState->parent == nullptr);
-    state->children.push_back (childState->item);
-    childState->parent = this;
+    std::lock_guard<std::recursive_mutex> _l(lock_);
```

Comment:
Don't use leading underscores for variable names. I know this name is not reserved in this case, but let's keep our naming convention consistent.